sa/_db/migrations/20170423163751_CreateAuthorizationsTable.sql

```diff
@@ -0,0 +1,31 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+CREATE TABLE `authorizations` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `token` mediumblob,
+  `identifierValue` varchar(255) NOT NULL,
+  `identifierType` int NOT NULL,
+  `registrationID` bigint(20) NOT NULL,
+  `status` int NOT NULL,
+  `expires` datetime NOT NULL,
+  `allowedChallenges` tinyint,
+  `validated` datetime NOT NULL,
+  `validationRecord` mediumblob,
+  `error` mediumblob,
+  `thumbprint` mediumblob,
+
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `identifier_key` (`identifierType`, `identifierValue`),
```

Comment:
Reminder: with the new keys, we no longer need this `UNIQUE KEY`. Also, FWIW, making this key `UNIQUE` would have caused problems anyhow because there can be multiple authorizations for the same identifier (for instance, renewals across time, or multiple accounts that can prove control of the same identifier).